components/Venue/venue.js

```diff
@@ -1,30 +1,34 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import Link from 'next/link';
 
 function Venue({ className, city }) {
 	return (
-		<div>
-			<div style={{'--image-url': `url(${city.img})`}} 
-			className='relative w-[300px] h-[400px] sm:w-[250px] sm:h-[350px] card-bg bg-[image:var(--image-url)] flex items-center justify-center p-4 cursor-pointer'>
+		<Link href={`/venue/${city.name}`}>
+			<div style={{'--image-url': `url(${city.img})`}}
+				className='relative w-[300px] h-[400px] sm:w-[250px] sm:h-[350px] card-bg bg-[image:var(--image-url)] flex items-center justify-center p-4 cursor-pointer'>
 				<div className='flex justify-between flex-col w-full h-full'>
 					<div className='flex justify-between items-center'>
 						<div><div className={`border text-white text-md rounded-lg p-1 text-center mt-2 ${city.cfp ? 'block': 'hidden' }`}>cfp is open</div></div>
```

Comment:
suggestion: instead of setting the classnames as `block` or `hidden`, we can render/remove the whole div from DOM if `city.cfp` exists/not exists.

Why? - If at some point, the Venue component is used 100 times in website, then we'll be saving 100 nodes from DOM.

Changes required-
- Removing `mt-2` from cfp div and adding it to parent. to keep everything aligned.
- setting ml-auto on map link to make sure it stays on right side even if cfp is not available.

@AceTheCreator you can suggest if we should do this or not.

Reply:
I agree with you @RamGoel 👍🏾